config/katello-answers.yaml

```diff
@@ -61,6 +61,7 @@ foreman::plugin::leapp: false
 foreman::plugin::monitoring: false
```

Comment:
Maybe add foreman::plugin::proxmox to foreman-answers.yaml, too? Its not releated to katello only.

Reply:
Unless you mean something else, it already is: https://github.com/theforeman/foreman-installer/pull/845/files